control_plane/attachment_service/src/reconciler.rs

```diff
@@ -410,7 +410,7 @@ impl Reconciler {
             match self.observed.locations.get(&node_id) {
                 Some(conf) if conf.conf.as_ref() == Some(&wanted_conf) => {
                     // Nothing to do
-                    tracing::info!("Observed configuration already correct.")
+                    tracing::info!(node_id=%node_id, "Observed configuration already correct.")
```

Comment:
```suggestion
                    tracing::info!(%node_id, "Observed configuration already correct.")
```
I think one can remove the `node_id=` from here

Reply:
Good point, there are a bunch of places that could benefit from that, I've added a commit.